etc/ardour5.profile

```diff
@@ -0,0 +1,37 @@
+# Firejail profile for ardour5
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/ardour5.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+
+noblacklist ${HOME}/.config/ardour4
+noblacklist ${HOME}/.config/ardour5
+noblacklist ${HOME}/.lv2
+noblacklist ${HOME}/.vst
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-passwdmgr.inc
+include /etc/firejail/disable-programs.inc
+
+caps.drop all
+ipc-namespace
+net none
+nodvd
+nogroups
```

Comment:
`nogroups` is potentially disruptive whenever it comes to professional audio, because we need to be a member of the `audio` group

Reply:
Yes, afaik audio is used for changing realtime scheduling. Fix this and others. Thank you.

Reply:
@SpotComms I wish to correct myself. Looks jack clients don't need to be in `audio` group, so ardour and lmm2 (but not mpd) can have `nogroups`. Sorry for the trouble.

Reply:
No problem, I've changed it back. So is the audio group only used for the little helper utility to switch between realtime and never by the actual application?

Reply:
I tried it out only briefly, and it looks the applications are fine with `nogroups`. Probably only the audio server itself, running outside the jail, must be a group member